bin/compile_dependencies.sh

```diff
@@ -1,45 +1,43 @@
-#!/bin/bash
-
+#!/bin/sh
 #
 # Compile the dependencies for production, CI and development.
 #
 # Usage, in the root of the project:
 #
 #     ./bin/compile_dependencies.sh
 #
-# Any extra flags/arguments passed to this wrapper script are passed down to pip-compile.
+# Any extra flags/arguments passed to this wrapper script are passed down to uv pip compile.
 # E.g. to update a package:
 #
 #     ./bin/compile_dependencies.sh --upgrade-package django
-
 set -ex
 
+command -v uv || (echo "uv not found on PATH. Install it https://astral.sh/uv" >&2 && exit 1)
+
+cwd="${PWD}"
```

Comment:
I don't think we need `cwd` if we use `toplevel` in the `pip-compile` commands (as `git rev-parse --show-toplevel` returns an absolute path), for example 
```
uv pip-compile \
   --output-file "$toplevel/requirements/base.txt" \
    "$@" \
    "$toplevel/requirements/base.in"
```

You could also call it something like `root_dir` to make the variables purpose more descriptive.

Reply:
that's a bit cleaner indeed, I'll change it